File: 1_add.js
// add numbers from 1 to n (including n)
function addUpTO(n){
    let total = 0;
    for(let i = 0;i<=n;i++){
        total += i;
    }
    return total;
}

var t1 = performance.now();
a = addUpTO(1000000000);
var t2 = performance.now();
console.log(a);
b = (t2-t1)/1000;
console.log("Time Elapsed:"+ b +" seconds");

File: practice/1_add.py
# add numbers from 1 to n (including n)
import timeit as t
def add(n):
    total = 0
    for i in range(n+1):
        total += i
    return total    

a = t.timeit()
d = add(6)    
b = t.timeit()
time = b-a
print(d)
print("Time Elapsed is {} seconds".format(time))